testah-junit/src/main/java/org/testah/runner/httpLoad/HttpActor.java

```diff
@@ -73,7 +75,11 @@ public ActorRef getWorkerRouter() {
     }
 
     public static List<ResponseDto> getResults(final Long hashId) {
-        return getResults().get(hashId);
+        HashMap<Long, List<ResponseDto>> results = getResults();
+        if (results.isEmpty() || !results.containsKey(hashId)) {
+            results.put(hashId, new ArrayList<ResponseDto>());
+        }
+        return results.get(hashId);
```

Comment:
make sure that getResults(final Long hashId) does not return null.
getResults() always returns a (possibly empty) HashMap.